test/e2e/storage/pvc_protection.go

```diff
@@ -45,14 +49,30 @@ var _ = utils.SIGDescribe("PVC Protection", func() {
 		nameSpace = f.Namespace.Name
 		framework.ExpectNoError(framework.WaitForAllNodesSchedulable(client, framework.TestContext.NodeSchedulableTimeout))
 
+		defaultSC := getDefaultStorageClass(client)
+		sc := defaultSC
+		scToDelete = nil
+		// Class having WaitForFirstConsumer means the PVC won't become Bound: create and use a class with Immediate
```

Comment:
Instead of creating a temporary storageclass for the test, can we just create a Pod and wait for the Pod to be running?  I want to get away from test cases creating special storageclasses because then we're not actually testing what has been installed.

Reply:
it complicates these pvc protection cases because they need control over when the pod(s) start running. e.g. the first case is that the bound pvc can be immediately deleted because there is no pod running

Reply:
For the first test case, the first step would be to delete the pod.

For the remaining two test cases, the first thing they already do is to create a pod, so that could be removed.

Reply:
:+1: working on it

Reply:
At some point we may want to add a test that verifies PV protection here and that too will require immediate binding mode. :-)

Reply:
There already are? https://sourcegraph.com/github.com/kubernetes/kubernetes/-/blob/test/e2e/storage/pv_protection.go

The reason why I prefer making the tests work with either mode is because I think that these test cases can be a good candidate for storage conformance tests, which should just use the default storageclass and not have to create its own storageclass.